api/envoy/config/bootstrap/v2/bootstrap.proto

```diff
@@ -148,6 +149,11 @@ message Bootstrap {
   // changes to this string, especially in multi-layer Envoy deployments or deployments using
   // extensions which are not upstream.
   string header_prefix = 18;
+
+  // Optional proxy version which will be used to set the value of :ref:`server.version statistic
+  // <server_statistics>` if specified. Envoy will not process this value, it will be sent as is to
+  // :ref:<stats sinks <envoy_api_msg_config.metrics.v2.StatsSink>.
+  google.protobuf.UInt64Value proxy_version = 19;
```

Comment:
@htuch thoughts on whether this should go into the `Node` message?

Reply:
Yeah. The `Node` message will eventually have a semantic version, but not one specified (it will come from the API, not Envoy build). There will also be a user agent string, which probably will include a git SHA in the case of Envoy.

I view the version here to be purely a stats convenience and not useful for anything in xDS. I think if we rename this to  `stats_server_version_override` then this sorts this problem out.